Build system tries to create a file on the system during the build step

```
$ make PREFIX=/usr
make -C fasm-plugin test
make -C xdc-plugin test
make[1]: Entering directory '/home/anubis/git/symbiflow-arch-pkgs/yosys-symbiflow-plugins-git/src/yosys-symbiflow-plugins/fasm-plugin'
make[1]: Entering directory '/home/anubis/git/symbiflow-arch-pkgs/yosys-symbiflow-plugins-git/src/yosys-symbiflow-plugins/xdc-plugin'
gcc -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -Wall -Wextra -ggdb -I/usr/share/yosys/include -MD -D_YOSYS_ -fPIC -I/usr/include -std=c++11 -Os -DYOSYS_ENABLE_READLINE -DYOSYS_ENABLE_PLUGINS -DYOSYS_ENABLE_GLOB -DYOSYS_ENABLE_ZLIB -DYOSYS_ENABLE_TCL -DYOSYS_ENABLE_ABC -DYOSYS_ENABLE_COVER -D_FORTIFY_SOURCE=2  -c -o fasm.o fasm.cc
gcc -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -Wall -Wextra -ggdb -I/usr/share/yosys/include -MD -D_YOSYS_ -fPIC -I/usr/include -std=c++11 -Os -DYOSYS_ENABLE_READLINE -DYOSYS_ENABLE_PLUGINS -DYOSYS_ENABLE_GLOB -DYOSYS_ENABLE_ZLIB -DYOSYS_ENABLE_TCL -DYOSYS_ENABLE_ABC -DYOSYS_ENABLE_COVER -D_FORTIFY_SOURCE=2  -c -o xdc.o xdc.cc
mkdir -p /usr/share/yosys/plugins/fasm_extra_modules/
mkdir: cannot create directory ‘/usr/share/yosys/plugins’: Permission denied
make[1]: *** [Makefile:18: install_modules] Error 1
make[1]: *** Waiting for unfinished jobs....
```

The default target is `install`: https://github.com/SymbiFlow/yosys-symbiflow-plugins/blob/master/Makefile#L7. Using `make plugins` instead avoids installation:

```
$ make PREFIX=/usr plugins
make -C fasm-plugin fasm.so
make[1]: Entering directory '/tmp/yosys-symbiflow-plugins/fasm-plugin'
gcc -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -g -fvar-tracking-assignments -fdebug-prefix-map=/build/yosys-git/src=/usr/src/debug -Wall -Wextra -ggdb -I/usr/share/yosys/include -MD -D_YOSYS_ -fPIC -I/usr/include -std=c++11 -Os -I/usr/include/python3.8 -I/usr/include/python3.8 -DWITH_PYTHON -DYOSYS_ENABLE_READLINE -DYOSYS_ENABLE_PLUGINS -DYOSYS_ENABLE_GLOB -DYOSYS_ENABLE_ZLIB -DYOSYS_ENABLE_TCL -DYOSYS_ENABLE_ABC -DYOSYS_ENABLE_COVER   -c -o fasm.o fasm.cc
gcc -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -g -fvar-tracking-assignments -fdebug-prefix-map=/build/yosys-git/src=/usr/src/debug -Wall -Wextra -ggdb -I/usr/share/yosys/include -MD -D_YOSYS_ -fPIC -I/usr/include -std=c++11 -Os -I/usr/include/python3.8 -I/usr/include/python3.8 -DWITH_PYTHON -DYOSYS_ENABLE_READLINE -DYOSYS_ENABLE_PLUGINS -DYOSYS_ENABLE_GLOB -DYOSYS_ENABLE_ZLIB -DYOSYS_ENABLE_TCL -DYOSYS_ENABLE_ABC -DYOSYS_ENABLE_COVER -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now -rdynamic -shared -o fasm.so fasm.o -lstdc++ -lm -lrt -lpython3.8 -lcrypt -lpthread -ldl -lutil -lm -lm -lboost_python38 -lboost_system -lboost_filesystem -lreadline -L/usr/lib/../lib -lffi -ldl -lz -ltcl8.6 -ltclstub8.6
make[1]: Leaving directory '/tmp/yosys-symbiflow-plugins/fasm-plugin'
make -C xdc-plugin xdc.so
make[1]: Entering directory '/tmp/yosys-symbiflow-plugins/xdc-plugin'
gcc -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -g -fvar-tracking-assignments -fdebug-prefix-map=/build/yosys-git/src=/usr/src/debug -Wall -Wextra -ggdb -I/usr/share/yosys/include -MD -D_YOSYS_ -fPIC -I/usr/include -std=c++11 -Os -I/usr/include/python3.8 -I/usr/include/python3.8 -DWITH_PYTHON -DYOSYS_ENABLE_READLINE -DYOSYS_ENABLE_PLUGINS -DYOSYS_ENABLE_GLOB -DYOSYS_ENABLE_ZLIB -DYOSYS_ENABLE_TCL -DYOSYS_ENABLE_ABC -DYOSYS_ENABLE_COVER   -c -o xdc.o xdc.cc
gcc -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -g -fvar-tracking-assignments -fdebug-prefix-map=/build/yosys-git/src=/usr/src/debug -Wall -Wextra -ggdb -I/usr/share/yosys/include -MD -D_YOSYS_ -fPIC -I/usr/include -std=c++11 -Os -I/usr/include/python3.8 -I/usr/include/python3.8 -DWITH_PYTHON -DYOSYS_ENABLE_READLINE -DYOSYS_ENABLE_PLUGINS -DYOSYS_ENABLE_GLOB -DYOSYS_ENABLE_ZLIB -DYOSYS_ENABLE_TCL -DYOSYS_ENABLE_ABC -DYOSYS_ENABLE_COVER -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now -rdynamic -shared -o xdc.so xdc.o -lstdc++ -lm -lrt -lpython3.8 -lcrypt -lpthread -ldl -lutil -lm -lm -lboost_python38 -lboost_system -lboost_filesystem -lreadline -L/usr/lib/../lib -lffi -ldl -lz -ltcl8.6 -ltclstub8.6
make[1]: Leaving directory '/tmp/yosys-symbiflow-plugins/xdc-plugin'

$ make PREFIX=/usr install
make -C fasm-plugin install
make[1]: Entering directory '/tmp/yosys-symbiflow-plugins/fasm-plugin'
mkdir -p /usr/share/yosys/plugins
cp fasm.so /usr/share/yosys/plugins/fasm.so
cp: cannot create regular file '/usr/share/yosys/plugins/fasm.so': Permission denied
make[1]: *** [Makefile:15: install] Error 1
make[1]: Leaving directory '/tmp/yosys-symbiflow-plugins/fasm-plugin'
make: *** [Makefile:23: install_fasm] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -4,7 +4,7 @@ PLUGINS_INSTALL := $(foreach plugin,$(PLUGIN_LIST),install_$(plugin))
 PLUGINS_CLEAN := $(foreach plugin,$(PLUGIN_LIST),clean_$(plugin))
 PLUGINS_TEST := $(foreach plugin,$(PLUGIN_LIST),test_$(plugin))
 
-all: install
+all: plugins
 
 define install_plugin =
 $(1).so:

--- a/fasm-plugin/Makefile
+++ b/fasm-plugin/Makefile
@@ -14,7 +14,7 @@ install: fasm.so
 	mkdir -p $(PLUGINS_DIR)
 	cp $< $(PLUGINS_DIR)/$<
 
-test: install
+test:
 	$(MAKE) -C tests all
 
 clean:

--- a/xdc-plugin/Makefile
+++ b/xdc-plugin/Makefile
@@ -18,7 +18,7 @@ install_modules: $(VERILOG_MODULES)
 	mkdir -p $(PLUGINS_DIR)/fasm_extra_modules/
 	cp $< $(PLUGINS_DIR)/fasm_extra_modules/$<
 
-test: install_plugin install_modules
+test:
 	$(MAKE) -C tests all
 
 .PHONY: install

